app/views/cost_of_living_landing_page/show.html.erb

```diff
@@ -28,54 +28,42 @@
   end
 %>
 
-<header class="cost-of-living-hub__header">
-  <%= render "shared/browse_header", { margin_bottom: 7, padding_top: 0 } do %>
-    <%= render 'govuk_publishing_components/components/breadcrumbs', {
-      breadcrumbs: breadcrumbs,
-      collapse_on_mobile: true,
-      inverse: true
-    } %>
+<%= render "shared/browse_breadcrumbs", {
+  breadcrumbs: breadcrumbs,
+  collapse_on_mobile: true,
+  inverse: true
+} %>
 
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
-        <%= render "govuk_publishing_components/components/title", {
-          title: content[:header][:heading],
-          inverse: true,
-          margin_top: 4,
-          margin_bottom: 6,
-        } %>
-      </div>
-    </div>
+<%= render "shared/browse_header", { margin_bottom: 7, padding_top: 0 } do %>
```

Comment:
Since we removed the two-thirds column, I think we need to pass `two_thirds: true` to the component here to match the width of the browse header markup.